docker-install/00-why_docker.md

```diff
@@ -0,0 +1,51 @@
+# Why Docker?
+### A brief intro to Docker as a data analysis tool
+
+For this workshop, we will be conducting all of our analyses in [Docker](https://www.docker.com/resources/what-container).
+Here we will introduce you to the strategy of using Docker for data analysis.
+
+Adapting from [this data science article](https://towardsdatascience.com/docker-for-data-scientists-5732501f0ba4) by Bobby Lindsey, have you ever heard or said these complaints?
+
+> “Not sure why it’s not working on your computer, it’s working on mine.”
+
+>“It’s a pain to install everything from scratch for Linux, Windows, and MacOS, and trying to build the same environment for each OS.”
```

Comment:
This does not seem like it would land with our intended audience.

Reply:
It's my least favorite quote: I shall delete it.